sips/sip-tutorial.md

```diff
@@ -3,21 +3,30 @@ layout: sip-landing
 title: Writing a new SIP
 ---
 
-This tutorial details of how to write a new SIP and adding it to the website.   Currently two mechanisms of providing a new SIP are recommended:
+This tutorial details of how to write a new SIP and adding it to the website.
 
-* Using Markdown
-* Using Google Docs.
+## How do I submit? ##
 
+The process to submit is simple:
 
-## Writing a SIP in Markdown ##
+* Fork the Scala documentation repository, [http://github.com/scala/scala.github.com](http://github.com/scala/scala.github.com).
+* Create a new SIP file in the `sips/pending/_posts/`. Use the [S(L)IP template](https://github.com/scala/slip/blob/master/slip-template.md)
+  * Make sure the new file follows the format:  `YYYY-MM-dd-{title}.md`.  Use the proposal date for `YYYY-MM-dd`.
+  * Use the [Markdown Syntax](http://daringfireball.net/projects/markdown/syntax) to write your SIP.
+  * Follow the instructions in the [README](https://github.com/scala/scala.github.com/blob/gh-pages/README.md) to build your SIP locally so you can ensure that it looks correct on the website.
+* Create a link to your SIP in the "pending sips" section of `index.md`
+* Commit your changes to your forked repository
```

Comment:
I would say "Commit your changes and push them to your forked repository".

But then do you need to also say "clone your fork" after forking?

I think it's safe to assume that anyone proposing language changes knows how to git/GitHub, so I would just keep it to the superficial facts, rather than the technical step-to-step.